src/erlcloud_cloudtrail.erl

```diff
@@ -115,6 +115,14 @@ describe_trails(Trails, Config) ->
     Json = [{<<"TrailNameList">>, [list_to_binary(T) || T <- Trails]}],
     ct_request("DescribeTrails", Json, Config).
 
+-spec describe_trails([string()], aws_config(), boolean()) -> ct_return().
```

Comment:
i would appreciate if we keep to the general api scheme where `Config` is the last parameter.
```erlang
spec describe_trails([string()], boolean(), aws_config()) -> ct_return().
```